Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/Features/Relationships/CanCreate/GET.feature

```diff
@@ -6,20 +6,23 @@ Feature: GET /Relationships/CanCreate
         When i1 sends a GET request to the /Relationships/CanCreate?peer={id} endpoint with id=i2.id
         Then the response status code is 200 (OK)
         And a Relationship can be established
+        And there is no code
 
     Scenario: Two identities with an active Relationship can't create another one
         Given Identities i1 and i2
         And an active Relationship r between i1 and i2
         When i1 sends a GET request to the /Relationships/CanCreate?peer={id} endpoint with id=i2.id
         Then the response status code is 200 (OK)
         And a Relationship can not be established
+        And the code is "error.platform.validation.relationship.relationshipToTargetAlreadyExists"
 
     Scenario: Two identities with a rejected Relationship can create one
         Given Identities i1 and i2
         And a rejected Relationship r between i1 and i2
         When i1 sends a GET request to the /Relationships/CanCreate?peer={id} endpoint with id=i2.id
         Then the response status code is 200 (OK)
         And a Relationship can be established
+        And there is no code
```

Comment:
I don't like that this is simply called `code`.

When we discuss things the "code" is usually the "status code". Strictly speaking this is `code` and that is `status code` so they are different, but I would still be more specific here so as to avoid confusion. For example, something like `there is no relationship creation check code` might do the trick.